features/steps/steps.py

```diff
@@ -44,3 +41,14 @@ def then_i_will_see_on_stdout(context):
 @then("I will see the following on stderr")
 def then_i_will_see_on_stderr(context):
     assert_that(context.process.stderr.strip(), equal_to(context.text))
+
+def get_user_spec(user_spec):
```

Comment:
I think the naming of this method is a little misleading. It _takes_ a `user_spec`, but it doesn't return one. Perhaps `get_command_prefix_for_user_spec` would be more accurate?

Reply:
Sure, I'll change it. Thanks